docs/file-handlers/index.md

```diff
@@ -137,8 +137,11 @@ The activation parameters will be lost if not persisted before this redirect occ
 
 You can see an example of using a data model object and handler method for caching the activation parameters in a cookie, in the [Markdown File Handler sample][markdown-file-handler].
```

Comment:
The `[markdown-file-handler]` reference needs a link associated with it.
There is no `[markdown-file-handler]` link defined later in the file.  While you have links defined for `[markdown-file-handler (C#)]` and `[markdown-file-handler (TypeScript)]` those are not the same name as `[markdown-file-handler]`. Could you please modify the `[markdown-file-handler]` reference name to point to either the C# or TypeScript examples?

Reply:
Got it, thanks. I will watch the build and see if things pass.